operators/src/mock/mock_raster_point_join.rs

```diff
@@ -91,23 +92,40 @@ impl VectorOperator for MockRasterPointJoinOperator {
         self: Box<Self>,
         context: &dyn crate::engine::ExecutionContext,
     ) -> Result<Box<crate::engine::InitializedVectorOperator>> {
-        InitializedOperatorImpl::create(
+        let vector_sources = self
+            .vector_sources
+            .into_iter()
+            .map(|o| o.initialize(context))
+            .collect::<Result<Vec<_>>>()?;
+
+        let raster_sources = self
+            .raster_sources
+            .into_iter()
+            .map(|o| o.initialize(context))
+            .collect::<Result<Vec<_>>>()?;
+
+        let result_descriptor = {
+            let mut columns = vector_sources[0].result_descriptor().columns.clone();
+            columns.insert(self.params.feature_name.clone(), FeatureDataType::Number);
+
+            VectorResultDescriptor {
+                spatial_reference: vector_sources.get(0).map_or_else(
+                    || SpatialReferenceOption::Unreferenced,
+                    |o| o.result_descriptor().spatial_reference,
+                ),
+                data_type: VectorDataType::MultiPoint,
```

Comment:
do we need to check that first vector source is actually MultiPoint?

Reply:
1b9537d